### ISSUE TITLE
Problems mounting Mac OS X shares 

### ISSUE THREAD

**Issue by kingosticks** [woutervanwijk/Pi-MusicBox#297]
Do we need to specify `sec=ntlmssp` when mounting shares from a server running Mac OS X as mentioned on http://opensource.com/life/15/3/pi-musicbox-guide ? If I had access to a max I'd check...

**Comment by jcass77** [comment]
Yes, that's the only way I've been able to get the shares working on my Mac.

**Comment by jodal** [comment]
I guess this is input to the docs.
